dashboard/src/shared/App.ts

```diff
@@ -19,11 +19,12 @@ export class App {
   public static async create(
     releaseName: string,
```

Comment:
So this shared directory is like a shared set of `models`?

Reply:
Yes, correct. We try to keep all the API related methods here and not in the actual actions.